src/App.vue

```diff
@@ -0,0 +1,64 @@
+      alt="user avatar"
+      class="header-avatar"
+    />
+  </header>
+
+  <RouterView class="routerView" />
+
+  <footer class="footer">
+    <p>© Copyright <b>2023</b></p>
+
+    <p><strong>Genesis front-end</strong></p>
+  </footer>
+</template>
+
+<script setup lang="ts">
+import { RouterView } from 'vue-router'
+import { getToken } from '@/api/index.js'
+import { onMounted } from 'vue'
+onMounted(async () => {
+  const token = await getToken()
+  document.cookie = `token=${token}`
+})
```

Comment:
I like you solution with cookies! It's interesting how you implemented it this way, because I wrote token auth

Reply:
Thank you : ))